sample/app/build.gradle

```diff
@@ -109,7 +109,7 @@ android {
             }
         }
 
-        task listDevices << {
```

Comment:
This is also updated because Gradle had a warning saying that the `leftShift` operator on Tasks API is deprecated.